### ISSUE TITLE
feat: Copy method name from timeline cell

### ISSUE THREAD

**Issue by tscottcertinia** [certinia/debug-log-analyzer#366]
### Is your feature request related to a problem?

I would like to be able to copy the method name from the tooltip in the timeline so that I can paste it into another document/or slack thread when discussing it.

### Describe the solution you'd like

Right click cell in the timeline and see a context menu to copy the method name.

### Describe alternatives you've considered

Navigate to the call stack and copy from there. (At present tho' if you copy from there you get the whole tree not just the text highlighted.

### Additional context

Slack me if you want to discuss. :)

**Comment by lcottercertinia** [comment]
@tscottcertinia 
You are right copy from call tree needs work, there is a ticket on the DB view to sort it out in general.
You can highlight text BUT do not release the mouse key then do ctrl+c, that will copy text from the row.
What is causing the issue is that the highlight is cleared on mouse up.

Copy from timeline is a good idea. We have a feature to add method names to the events on the timeline too.
